src/librustdoc/formats/renderer.rs

```diff
@@ -86,8 +86,6 @@ pub(crate) fn run_format<'tcx, T: FormatRenderer<'tcx>>(
             }
 
             cx.mod_item_out()?;
-        // FIXME: checking `item.name.is_some()` is very implicit and leads to lots of special
```

Comment:
This FIXME is still applicable I think. This was changed to be a match in https://github.com/rust-lang/rust/commit/1e55fc124382ab0fde9ac151b437bddb7b280554, but the broader point (about checking if the name is present being bad) still applies